documentation/src/cli.md

```diff
@@ -33,10 +33,24 @@ forest-cli --token eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJBbGxvdyI6WyJyZWFkIiwi
 ```
 
 When the Forest daemon is started, an admin token will be displayed (alternatively, use `--save-token <token>` flag to save it on disk).
+## Sending filecoin tokens (money) from your wallet.
+
+For sending filecoin token, forest daemon must be running. You can do so by running:
+
+`forest --chain calibnet --target-peer-count 50 --encrypt-keystore false`
```

Comment:
Do we need this peer count and disable the keystone? I mean, it's OK for the CI and testing, but in general, we should recommend and default to have it encrypted.